Compilation of generated classes seem to fail

I added all the expected packages from `killbill-api`, i.e:

```
  <acceptedPackages>
    <package>org.killbill.billing</package>
    <package>org.killbill.billing.tenant</package>
    <package>org.killbill.billing.tenant.api</package>
    <package>org.killbill.billing.util</package>
    <package>org.killbill.billing.util.customfield</package>
    <package>org.killbill.billing.util.nodes</package>
    <package>org.killbill.billing.util.entity</package>
    <package>org.killbill.billing.util.audit</package>
    <package>org.killbill.billing.util.queue</package>
    <package>org.killbill.billing.util.callcontext</package>
    <package>org.killbill.billing.util.api</package>
    <package>org.killbill.billing.util.tag</package>
    <package>org.killbill.billing.security</package>
    <package>org.killbill.billing.security.api</package>
    <package>org.killbill.billing.payment</package>
    <package>org.killbill.billing.payment.plugin</package>
    <package>org.killbill.billing.payment.plugin.api</package>
    <package>org.killbill.billing.payment.api</package>
    <package>org.killbill.billing.catalog</package>
    <package>org.killbill.billing.catalog.api</package>
    <package>org.killbill.billing.catalog.api.rules</package>
    <package>org.killbill.billing.osgi</package>
    <package>org.killbill.billing.osgi.api</package>
    <package>org.killbill.billing.osgi.api.config</package>
    <package>org.killbill.billing.usage</package>
    <package>org.killbill.billing.usage.api</package>
    <package>org.killbill.billing.account</package>
    <package>org.killbill.billing.account.api</package>
    <package>org.killbill.billing.entitlement</package>
    <package>org.killbill.billing.entitlement.api</package>
    <package>org.killbill.billing.currency</package>
    <package>org.killbill.billing.currency.api</package>
    <package>org.killbill.billing.overdue</package>
    <package>org.killbill.billing.overdue.api</package>
    <package>org.killbill.billing.invoice</package>
    <package>org.killbill.billing.invoice.api</package>
    <package>org.killbill.billing.invoice.api.formatters</package>    
  </acceptedPackages>
```

I was able to generate the classes 🎉 but then when I compile the code, I get the following error:

```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /private/tmp/output/project/src/main/java/org/killbill/billing/entitlement/api/boilerplate/BlockingStateImp.java:[185,16] method hashCode() is already defined in class org.killbill.billing.entitlement.api.boilerplate.BlockingStateImp
[ERROR] /private/tmp/output/project/src/main/java/org/killbill/billing/entitlement/api/boilerplate/BlockingStateImp.java:[202,19] method toString() is already defined in class org.killbill.billing.entitlement.api.boilerplate.BlockingStateImp
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.513 s
[INFO] Finished at: 2022-02-17T16:24:13-08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project project: Compilation failure: Compilation failure: 
[ERROR] /private/tmp/output/project/src/main/java/org/killbill/billing/entitlement/api/boilerplate/BlockingStateImp.java:[185,16] method hashCode() is already defined in class org.killbill.billing.entitlement.api.boilerplate.BlockingStateImp
[ERROR] /private/tmp/output/project/src/main/java/org/killbill/billing/entitlement/api/boilerplate/BlockingStateImp.java:[202,19] method toString() is already defined in class org.killbill.billing.entitlement.api.boilerplate.BlockingStateImp
[ERROR] -> [Help 1]

```

--- a/src/main/java/org/killbill/billing/tool/pojogen/Implementation.java
+++ b/src/main/java/org/killbill/billing/tool/pojogen/Implementation.java
@@ -20,6 +20,7 @@ import java.math.BigInteger;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
@@ -56,6 +57,8 @@ import com.github.javaparser.utils.SourceRoot;
 public class  Implementation {
 
   private static final Log log = new Log(Implementation.class);
+  private static final Set<String> skip = new HashSet<String>(
+      Arrays.asList( "hashCode()", "toString()", "equals(java.lang.Object)"));
 
   private final Mapping mapping;
   private final ResolvedInterfaceDeclaration declaration;
@@ -67,7 +70,7 @@ public class  Implementation {
   private final List<Property> properties;
   private final List<String> imports;
 
-  private Implementation( ResolvedInterfaceDeclaration declaration, String namespace, String name, String base,
+  private Implementation(ResolvedInterfaceDeclaration declaration, String namespace, String name, String base,
       List<String> imports, List<Property> properties, List<Method> methods, Mapping mapping, Symbols symbols){
     this.declaration = declaration;
     this.namespace = namespace;
@@ -187,12 +190,13 @@ public class  Implementation {
     Symbols symbols, String namespace, String name, 
     ResolvedInterfaceDeclaration declaration){
     symbols = new Symbols(symbols);
-    List<MethodUsage> usages = Method.unique(traverse(declaration));
+    List<MethodUsage> usages = filter(Method.unique(traverse(declaration)));
     for(MethodUsage usage: usages){
       symbols.add(usage.getName());
     }
     Importer importer  = new Importer(symbols, namespace, name);
     importer.add(namespace, Namespaces.join(name, "Builder"));
+
     importer.add("java.lang", "java.lang.Boolean");
     importer.add("java.lang", "java.lang.Byte");
     importer.add("java.lang", "java.lang.Character");
@@ -209,9 +213,9 @@ public class  Implementation {
     importer.add("java.io"  , "java.io.Serializable");
     importer.add("java.util", "java.util.Objects");
     importer.add("java.util", "java.util.Arrays");
+
     importer.add(declaration.getPackageName() , declaration.getQualifiedName());
     ArrayList<ResolvedType> types = new ArrayList<ResolvedType>();
-
     for(MethodUsage usage : usages){
       gather(usage, types);
     }
@@ -220,6 +224,7 @@ public class  Implementation {
     Mapping mapping  = importer.getMapping();
     symbols =  importer.getSymbols();
     HashMap<String,String> fields = new  HashMap<String,String>();
+
     for(String id: Property.possible(usages)){
       if(!fields.containsKey(id)){
         int attempts = 0;
@@ -265,6 +270,16 @@ public class  Implementation {
         parameter.addModifier(Modifier.FINAL);
       }
     }
+  }
+  private static List<MethodUsage> filter(List<MethodUsage> usages){
+    if(usages != null){
+      for(int i = usages.size() - 1 ; i >= 0 ; i--){
+        if(skip.contains(Method.getSignature(usages.get(i)))){
+          usages.remove(i);
+        }
+      }
+    }
+    return usages;
   }
   private static List<MethodUsage> traverse(ResolvedInterfaceDeclaration declaration){
     ArrayList<MethodUsage> usages = new ArrayList<MethodUsage>();

